fasttrips/Passenger.py

```diff
@@ -509,7 +509,10 @@ def setup_passenger_pathsets(self, iteration, stops, trip_id_df, trips_df, modes
         `pf_iteration`                int64  iteration in which these paths were found
         `pathnum`                     int64  the path number for the path within the pathset
         `pf_cost`                   float64  the cost of the entire path
+        `pf_fare`                   float64  the fare of the entire path
         `pf_probability`            float64  the probability of the path
+        `pf_initcost`               float64  the initial cost of the entire path
```

Comment:
"INIT" something in this case is sort of like "FINAL" before "FINAL FINAL".  It might be good at some point to change this { and its siblings } to be named after their stage.  I.e. "Cost for Pathfinding" rather than "initial cost".